alpenhorn/service.py

```diff
@@ -32,6 +34,9 @@ def cli():
     # Load the configuration for alpenhorn
     config.load_config()
 
+    # Set up logging
+    logger.start_logging()
+
```

Comment:
The logging isn't really finalised for the client at the moment. We need to decide how much logging should be displayed for the client, and how it should be configured.